RInAppMessaging/Classes/Views/Presenters/FullViewPresenter.swift

```diff
@@ -57,9 +57,8 @@ internal class FullViewPresenter: BaseViewPresenter, FullViewPresenterType {
         sendImpressions()
 
         if let unwrappedUri = sender.uri {
-            guard let uriToOpen = URL(string: unwrappedUri),
-                UIApplication.shared.canOpenURL(uriToOpen) else {
-
+            
+            guard let uriToOpen = URL(string: unwrappedUri) else {
```

Comment:
Because it fails for https url's on some iOS 14 versions (14.2 for example) where the default browser is Chrome.
And also because if you want to use a deeplink as a url in campaign button, you need to update LSApplicationQueriesSchemes in the app first (If you already released an app you would need to release again).